### PR TITLE
Control dispatch policy explicit

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst  PR

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/vespa/model/content/cluster/ClusterTest.java [hunk 1]
```diff
@@ -57,6 +58,21 @@ public void requireThatSearchCoverageIsApplied() throws ParseException {
...
+            assertTrue(config.dataset(0).useroundrobinforfixedrow());
+        }
+    }
+
+    @Test
+    public void requireThatDispatchTuningIsApplied() throws ParseException {
+        ContentCluster cluster = newContentCluster(joinLines("<search>", "</search>"),
+                joinLines("<tuning>",
+                        "</tuning>"));
+        for (Dispatch tld : cluster.getSearch().getIndexed().getTLDs()) {
```
Comment (geirst) [inline config-model/src/test/java/com/yahoo/vespa/model/content/cluster/ClusterTest.java]: Consider adding an assert for the number of dispatchers to ensure that we actually check something in the loop.

**Review by geirst — APPROVED** [review]
Overall looks good, please have a look at my comment

File: config-model/src/test/java/com/yahoo/vespa/model/content/cluster/ClusterTest.java (hunk 1 above)
Comment (baldersheim) [reply config-model/src/test/java/com/yahoo/vespa/model/content/cluster/ClusterTest.java]: Good point. It turned out that a few tests did not test anything.

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**